config/tests/guest/libvirt/backuprestore.cfg

```diff
@@ -0,0 +1,214 @@
+include tests-shared.cfg
+username = root
+password = 123456
+main_vm = virt-tests-vm1
+vms = virt-tests-vm1
+#Network
+nettype = bridge
+netdst=virbr0
+# Using Text mode of installation
+#display = 'nographic'
+display = 'vnc'
+take_regular_screendumps = no
+keep_screendumps_on_error = no
+keep_screendumps = no
+store_vm_register = no
+virt_install_binary = /usr/bin/virt-install
+qemu_img_binary = /usr/bin/qemu-img
+hvm_or_pv = hvm
+machine_type = pseries
+only bridge
+no xen, lxc, esx, ovmf
+#Filterout unwanted disk types
+no ide,xenblk,lsi_scsi,ahci,sd
+no spapr-vlan
+no qed,qcow2v3,raw_dd,vmdk, usb2
+no e1000-82540em,e1000-82545em,e1000-82544gc,xennet,nic_custom
+only no_virtio_rng
+only smp2
+only no_9p_export
+only no_pci_assignable
+only (image_backend=filesystem)
+only smallpages
+smp = 32
+vcpu_cores = 32
+vcpu_threads = 1
+vcpu_sockets = 1
+# 32G
+mem = 32768
+
+variants:
+    - guest_import:
+        only unattended_install.import.import.default_install.aio_native
+
+    - guest_backuprestore:
+        only virsh.snapshot_dumpxml,virsh.snapshot_par_cur,virsh.snapshot_disk,virsh.snapshot_create_as
+        #Skipping qed 98 test cases ,as these are supported only for libvirt having version 1.1.0
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.attach_img_qed.snapshot_from_xml.disk_external.sys_checkpoint.default.memory_external
```

Comment:
you could filter all qed tests with something like 
```
no virsh.snapshot_disk..attach_img_qed
```
it should match all tests